group_vars/all

```diff
@@ -7,3 +7,26 @@ ad_domain: "{{ krb5_realm.lower() }}"
 computer_ou: ou=Hosts,ou=morhidi,ou=HadoopClusters,ou=morhidi,dc=ad,dc=sec,dc=cloudera,dc=com
 domain: vpc.cloudera.com
 kdc: w2k8-1.ad.sec.cloudera.com
+
+# Only the latest CDH minor version is available in the official repo
```

Comment:
Which official repo does this refer to? All CDH versions can be found in archive.cloudera.com.